packages/PHPStanStaticTypeMapper/TypeMapper/ClosureTypeMapper.php

```diff
@@ -43,7 +43,15 @@ public function mapToPHPStanPhpDocTypeNode(Type $type, string $typeKind): TypeNo
             $typeKind
         );
 
-        return new SpacingAwareCallableTypeNode($identifierTypeNode, [], $returnDocTypeNode);
+        $parameterDocTypeNodes = [];
+        foreach ($type->getParameters() as $parameter) {
+            $parameterDocTypeNodes[] = $this->phpStanStaticTypeMapper->mapToPHPStanPhpDocTypeNode(
+                $parameter->getType(),
+                $typeKind
+            );
+        }
+
+        return new SpacingAwareCallableTypeNode($identifierTypeNode, $parameterDocTypeNodes, $returnDocTypeNode);
```

Comment:
could you move it to private method and call it in return? that's should solve cognitive complexity phpstan notice